index.html

```diff
@@ -241,5 +241,35 @@ <h2>Examples</h2>
         ...
       </pre>
     </section>
+    <section class="informative">
+      <h2>Privacy and Security Considerations</h2>
+      <p>
+        The mechanisms introduced by this specification are only available to an application which
+        has called {{MediaDevices/getDisplayMedia()}}, and where the user selected to capture some
+        [=display surface=] <var>S</var>. This means that the capturing application already has
+        access to all of the pixels visible on <var>S</var>.
+      </p>
+      <p>
+        If the capturing application so chooses, it can include the cursor among those pixels by
+        specifying the {{CursorCaptureConstraint}} {{CursorCaptureConstraint/"always"}}. This can
+        either be done when calling {{MediaDevices/getDisplayMedia()}}, or through
+        {{MediaStreamTrack/applyConstraints()}}. If {{MediaStreamTrack/applyConstraints()}} is used,
+        the cursor constraint might even be applied to a clone of the original track.
+      </p>
+      <p>
+        It follows that the mechanisms currently included in this specification do NOT change what
+        information is available to a capturing application. These mechanisms only make access to
+        this information cheaper and more reliable, by removing the need for the capturing
+        application to heuristically scan for the cursor's position in the frames.
+      </p>
+      <p>
+        When implementing this feature, care must be taken so that the cursor's location would not
+        be exposed in situations where it would not be drawn in the frame. For example, if the
```

Comment:
In the other PR, I'm also mentioning the complementary case "send (-1, -1) only when the cursor disappears". An example of incorrect behavior would be to send "(-1, -1)" for subsequent moves outside the surface (i.e. without entering again) which would reveal that the mouse continues to move, something you cannot know by just scanning the frames.

Reply:
I think the spec could stand to mention that a new event should not be fired if none of its values has changed. We can follow up with that separately.